cms/djangoapps/contentstore/rest_api/v2/views/home.py

```diff
@@ -0,0 +1,91 @@
+"""HomePageCoursesViewV2 APIView for getting content available to the logged in user."""
+import edx_api_doc_tools as apidocs
+from rest_framework.request import Request
+from rest_framework.response import Response
+from rest_framework.views import APIView
+from openedx.core.lib.api.view_utils import view_auth_classes
+
+from cms.djangoapps.contentstore.utils import get_course_context_v2
+from cms.djangoapps.contentstore.rest_api.v2.serializers import CourseHomeTabSerializerV2
+
+
+@view_auth_classes(is_authenticated=True)
+class HomePageCoursesViewV2(APIView):
+    """View for getting all courses available to the logged in user."""
+
+    @apidocs.schema(
+        parameters=[
+            apidocs.string_parameter(
+                "org",
+                apidocs.ParameterLocation.QUERY,
+                description="Query param to filter by course org",
+            ),
+            apidocs.string_parameter(
+                "search",
+                apidocs.ParameterLocation.QUERY,
+                description="Query param to filter by course name, org, or number",
+            ),
+            apidocs.string_parameter(
```

Comment:
What are the valid values of this parameter?

Reply:
^ I replied here: https://github.com/openedx/edx-platform/pull/34173#discussion_r1479730737